ui/src/components/Employee/Create/index.tsx

```diff
@@ -0,0 +1,160 @@
+          <div className={c('title-container')}>
+            <h2>{p.values ? p.values.name : 'Create Employee'} </h2>
+            <span className={c('sub-title')}>Manage your employee</span>
+          </div>
+
+          <div className={c('email-container', 'field-container')}>
+            <span className={c('field-title')}>Email</span>
+            <TextField placeholder="Enter email" type="email" name="email" />
+          </div>
+          <div className={c('name-container', 'field-container')}>
+            <span className={c('field-title')}>Name</span>
+            <TextField placeholder="Enter name" type="text" name="name" />
+          </div>
+
+          <div className={c('logo', 'field-container')}>
+            <span className={c('field-title')}>Head Shot</span>
+            <ImageUploadField className={c('image-container')} type={'file'} name="headshot" />
+          </div>
+
+          <div className={c('right-container')}>
+            <div className={c('ecode-container', 'field-container')}>
+              <span className={c('field-title')}>E-Code</span>
+              <TextField placeholder="Enter e-code" type="text" name="ecode" />
+            </div>
+
+            <div className={c('designation-container', 'field-container')}>
+              <span className={c('field-title')}>Designation</span>
+              <SelectField
+                className={c('select-field')}
+                name="designation_id"
+                options={p.designations.map((d) => ({ id: d.id, name: d.name }))}
+                isLoading={p.isFetchingDesignations}
+                isLogoHidden={true}
+              />
+            </div>
+            <div className={c('org-container', 'field-container')}>
+              <span className={c('field-title')}>Organization</span>
+              <SelectField
+                className={c('select-field')}
+                name="organization_id"
+                options={p.organizations.map((o) => ({ ...o, logo: o.square_logo }))}
+                isLoading={p.isFetchingOrgs}
+              />
+            </div>
+          </div>
+
+          <button className={c('save-button')} type="submit" disabled={p.isSubmitting}>
+            <div className={c({ 'saving-in': p.isSubmitting })}>
+              {p.isSubmitting ? 'Saving...' : 'Save'}
+              <span />
+            </div>
+          </button>
+        </form>
+      </div>
+    </div>
+  );
+};
+
+// ---
+
+export interface CreateEmployeeFormValues {
+  email: string;
+  name: string;
+  ecode: string;
+  headshot: string;
+  designation_id: string;
+  organization_id: string;
+}
+
+const validationSchema = Yup.object().shape({
+  designation_id: Yup.string().required('Required'),
+  ecode: Yup.string().required('Required'),
+  email: Yup.string().email().required('Required'),
+  headshot: Yup.string(),
+  name: Yup.string()
+    .min(2, 'Must be 2 characters or more')
+    .max(16, 'Must be 16 characters or less')
```

Comment:
Let's make it 100. There are people with absurdly long names. Our goal is just to provide a useful validation to protect sending too big a payload to server directly from UI.